hexmap.py

```diff
@@ -63,7 +83,7 @@ def blank_map():
     for column in range(0, 9):
         starmap[column] = {}
         for row in range(0, 11):
-            starmap[column][row] = World(" ", " ", " ", " ", "_", "       ")
+            starmap[column][row] = None

```

Comment:
If you leave this as in instance of `World`, then later you could call `printmap[row][column] = starmap[column][row].print_lines()` and move all the `print_map.append` logic into this class, and then you don't need the `if starmap[column][row] is not None:`.

```python
class World2:
    def print_lines(self):
        return [
            f"  {self.starport} {self.gas_giants}",
            f"{starmap[column][row].naval_base()}  {starmap[column][row].worldtype()}   "
            # etc.
        ]
```